docs/editor/terminal-shell-integration.md

```diff
@@ -0,0 +1,173 @@
+---
+Order: 12
+Area: editor
+TOCTitle: Terminal Shell Integration
+ContentId: a6a1652b-c0d8-4054-a2da-feb915eef2cc
+PageTitle: Terminal Shell Integration in Visual Studio Code
+DateApproved: 6/9/2022
+MetaDescription: Visual Studio Code's integrated terminal has the ability to integration with various shells, enhancing the capabilities of the terminal.
+---
+
+# Terminal Shell Integration
+
+Visual Studio Code has the ability to integrate with common shells, allowing the terminal to understand more about what's actually happening inside the shell. This additional information enables some [compelling features](#_features) such as command/directory detection, command decorations and command navigation.
```

Comment:
```suggestion
Visual Studio Code has the ability to integrate with common shells, allowing the terminal to understand more about what's actually happening inside the shell. This additional information enables some [compelling features](#_features) such as command/directory detection, point of interest decorations and command navigation.
```

Reply:
ok how about `such as directory detection and command detection, decorations, and navigation`?

Reply:
I improved the wording, good idea since it's one of the first things in the page 👍